api.planx.uk/admin/session/oneAppXML.test.ts

```diff
@@ -0,0 +1,62 @@
+import supertest from "supertest";
+import app from "../../server";
+import { queryMock } from "../../tests/graphqlQueryMock";
+import { authHeader } from "../../tests/mockJWT";
+
+const endpoint = (strings: TemplateStringsArray) => `/admin/session/${strings[0]}/xml`;
+
+describe("Download feedback CSV endpoint", () => {
```

Comment:
nit: this description is still copied from the other endpoint!

Reply:
🤦